### PR TITLE
JSI-6: Add test section and update it to allow TDD

### PR DESCRIPTION
**Pull request by drossi-sa**

### CODE REVIEW

File: build.gradle [hunk 1]
```diff
@@ -91,4 +91,4 @@ test {
     useJUnitPlatform()
 }
 
-test.finalizedBy( jacocoTestReport )
+//test.finalizedBy( jacocoTestReport )
```
Comment (mhuberSA) [inline build.gradle:94]: Please revert this.

File: src/test/java/org/secureauth/sarestapi/util/RestApiHeaderTest.java [hunk 2]
```diff
@@ -24,7 +24,7 @@ public void setup() {
 		saAuth = new SAAuth(applicationID,applicationKey,realm);
 	}
 
-	@Test
```
Comment (mhuberSA) [inline src/test/java/org/secureauth/sarestapi/util/RestApiHeaderTest.java:27]: Also revert this remove lines.

File: .idea/compiler.xml [hunk 3]
```diff
@@ -0,0 +1,16 @@
...
+        <processorPath useClasspath="false">
+          <entry name="$USER_HOME$/.gradle/caches/modules-2/files-2.1/org.apache.logging.log4j/log4j-core/2.5/7ed845de1dfe070d43511fab321784e6c4118398/log4j-core-2.5.jar" />
+          <entry name="$USER_HOME$/.gradle/caches/modules-2/files-2.1/org.apache.logging.log4j/log4j-api/2.5/e7fd981408caba8a0c0fb276413562468d260160/log4j-api-2.5.jar" />
+        </processorPath>
+        <module name="saRestApi.main" />
+      </profile>
+    </annotationProcessing>
+    <bytecodeTargetLevel target="11" />
+  </component>
+</project>
```
Comment (mhuberSA) [inline .idea/compiler.xml:16]: we should not add the .idea files.

**Comment by drossi-sa** [comment]
I will send a new pr without the idea files commited

**Pull request closed without merging**